### ISSUE TITLE
Kyma CLI - Deployment of function ignores environment variables in config.yaml

### ISSUE THREAD

**Issue by lechnerc77** [kyma-project/cli#715]
**Description**

In the discussion around the issue [10311](https://github.com/kyma-project/kyma/issues/10311) in the Kyma repository the option to deploy functions via Kyma CLI was highlighted including the option to pass environment variables in the process

I know tried this option but the CLI seems to ignore the environment variables defined in the _config.yaml_ file.

The rest of the function i.e. the source code and the dependencies are deployed correctly taking into account the settings in the config file. 

Kyma CLI version: 1.18.0
Kyma Version: 1.17 (SAP Coud Platform Trial - managed Kyma)

**Expected result**

When defining environment variables in the _config.yaml_ file and applying the configuration via `kyma apply function` the corresponding configuration i.e. the variables should be deployed.

**Actual result**

The Kyma CLI just deploys the function code and dependencies but ignores the environment variables defined in the config.yaml file.

**Steps to reproduce**

- Clone the repository [https://github.com/lechnerc77/kymaclibug_env](https://github.com/lechnerc77/kymaclibug_env)
- Create a namsespac `function-with-envar`
- Deploy the configmap and the secrets located in the directory _baseconf_ to this namespace
    ```bash
    kubectl -n function-with-envar apply -f ./baseconf/configmap.yaml
    kubectl -n function-with-envar apply -f ./baseconf/secrets.yaml
    ```
- Deploy the function after adjusting the path to the source in the `config.yaml` file via  
    ```bash
     kyma apply function
    ```
- Check the deployed function via `kubectl -n function-with-envar describe function function-with-env`
Result: The environment variables defined in the file _config.yaml_ are not part of the deployed function
- Alternative: Execute `kyma apply function --dry-run -o yaml` to get insight into the yaml that omits the `env` section in the _config.yaml_ file

**Troubleshooting**

--

**Comment by kwiatekus** [comment]
HI @lechnerc77 
This feature was added to master quite recently (https://github.com/kyma-project/kyma/issues/10057)
It is not supported in CLI version 1.18.0.
1.19.0 is on the way.  [rc1](https://github.com/kyma-project/cli/releases/tag/1.19.0-rc1) is out already

**Comment by lechnerc77** [comment]
Hi @kwiatekus,
I confirm: with release 1.19 rc1 the deployment including the specified environment variables works as expected. Thanks for the hint Looking forward to the final release :-)

**Comment by kwiatekus** [comment]
Fixed with 1.19. Closing
